lib/checks/canFindRoots.js

```diff
@@ -0,0 +1,39 @@
+const Node = require('../node');
+const resolvesToConstant = require('./resolvesToConstant.js');
+
+function canFindRoots(equation) {
```

Comment:
Can you move your comment below to the top of the function instead, as a docstring? Make it be like "Returns true if the equation is of the form ..."